starter-code/views/index.ejs

```diff
@@ -7,6 +7,11 @@
   <body>
     <h1><%= title %></h1>
     <p>Welcome to <%= title %></p>
+    <canvas id="myChart" width="400" height="400"></canvas>
+
   </body>
+  
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
   <script src="/javascripts/financial-data.js" charset="utf-8"></script>
-</html>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
```

Comment:
Since you are using `window.onload` doesn't matter the script's order, but is a good practice put it where they should be. 
In your `financial-data.js` you are using the `Chart.bundle.js` so `financial-data.js` must be the last one in the html